alibi_detect/cd/pytorch/mmd.py

```diff
@@ -117,7 +117,8 @@ def score(self, x: Union[np.ndarray, list]) -> Tuple[float, float, float]:
         x_ref = torch.from_numpy(x_ref).to(self.device)  # type: ignore[assignment]
         x = torch.from_numpy(x).to(self.device)  # type: ignore[assignment]
         # compute kernel matrix, MMD^2 and apply permutation test using the kernel matrix
-        n = x.shape[0]
+        # TODO: (See https://github.com/SeldonIO/alibi-detect/issues/540)
+        n = x.shape[0]  # type: ignore
```

Comment:
**Note**: This error is discussed here: https://github.com/SeldonIO/alibi-detect/issues/540